lobby/src/main/java/org/triplea/server/ServerConfiguration.java

```diff
@@ -36,4 +48,15 @@ private static CreateIssueStrategy createIssueStrategy() {
         .allowErrorReport(new ErrorReportGateKeeper())
         .build();
   }
+
+
+  private static Function<RegisteredUserLoginRequest, LobbyLoginResponse> registeredUserLoginStrategy() {
+    // TODO: stubbed value, implement this;
+    return loginRequset -> LobbyLoginResponse.newFailResponse("stubbed response");
```

Comment:
I believe you spelled loginRequest wrong here
```suggestion
    return loginRequest -> LobbyLoginResponse.newFailResponse("stubbed response");
```